routers/user/auth.go

```diff
@@ -1,5 +1,4 @@
 // Copyright 2014 The Gogs Authors. All rights reserved.
-// Copyright 2018 The Gitea Authors. All rights reserved.
```

Comment:
Why remove copyright?

Reply:
Oops, my bad... Must be when I was resolving conflicts

Reply:
Fixed @techknowlogick